src/modules/hosts/host-list/host-list.js

```diff
@@ -46,6 +47,11 @@
             }, 1000 * config.nodeRefreshIntervalTime, 1);
         }
 
+        function redirectToGrafana(host, $event){
+            utils.redirectToGrafana("hosts", $event, {clusterId: vm.clusterId,
+                                                      hostName: host.name.split(".").join("_")});
+        }
```

Comment:
@cloudbehl I think its not correctly indented. Whole statement should be in one line.